dspback/routers/discovery.py

```diff
@@ -84,14 +84,50 @@ async def search(
                 }
             },
         )
+        stages[0]['$search']['count'] = {'type': 'total'}
+        stages.append({'$project': {'meta': "$$SEARCH_META",
```

Comment:
I don't think this approach will work because we are further filtering out results with a minimum score threshold after this stage, but the metadata from the 'search' stage will not account for that.